rcl_interfaces/msg/FloatingPointRange.msg

```diff
@@ -1,14 +1,13 @@
 # Represents bounds and a step value for a floating point typed parameter.
 
-# Lower bound for valid values, inclusive.
-double minimum_value
+# Start value for valid values range, inclusive.
```

Comment:
This looks weird to me, maybe instead:

> Start value for valid values in the range, inclusive.

Or maybe even better:

> Start value for the range of valid values, inclusive.

Honestly the original:

> Start value for valid values, inclusive.

Was probably the best.